metaswitch/common/network_namespace.py

```diff
@@ -1,4 +1,4 @@
-# @file alarms.py
+# @file network_namespace.py
 #
```

Comment:
Hmm, I'm actually going to back down on this one. I don't know where else this is used and might easily break something without running the UTs. If you aren't happy with this, I can raise a low priority issue.